src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -404,6 +421,41 @@ def _get_linux_fx_version(cmd, resource_group_name, name, slot=None):
     return site_config.linux_fx_version
 
 
+def url_validator(url):
+    try:
+        result = urlparse(url)
+        return all([result.scheme, result.netloc, result.path])
+    except:
+        return False
+
+
+def _get_linux_multicontainer_decoded_config(cmd, resource_group_name, name, slot=None):
+    from base64 import b64decode
+    linux_fx_version = _get_linux_fx_version(cmd,resource_group_namenameslot)
+    if not any([linux_fx_version.startswith(s for s in MULTI_CONTAINER_TYPES)]):
+        raise CLIError("Cannot decode config that is not one of the"
+                       " following types: {}".form    (','(MULTI_CONTAINER_TYPES)))
+    try:
+        return b64decode(linux_fx_version.spl('|'[1].encode('utf-8')))
+    except:
+        raise CLIError('Could not decode config')
+
+
+def _get_linux_multicontainer_encoded_config_from_file(cmd, resource_group_name, name, file_name, slot=None):
+     from base64 import b64encode
+     config_file_bytes = None
+     if url_validator(file_name):
+         from urllib.request import urlopen
+         response = urlopen(file_name, context=_ssl_context())
+         config_file_bytes = response.read()
+     else:
+         with open(file_name) as f:
```

Comment:
you can read the file as binary, so you don't need to encode again in the 2 line below